CoffeeChat/Controller/ConnectSocialMediaViewController.swift

```diff
@@ -0,0 +1,151 @@
+        self.view.insertSubview(backgroundImage, at: 0)
+        
+        title = "Social Media"
+        
+        infoTextView.text = "Help your Pear contact you by connecting your social media accounts. Your information will only be shared with the verified Cornell students you are paired with."
+        infoTextView.textAlignment = .center
+        infoTextView.textColor = .greenGray
+        infoTextView.backgroundColor = .clear
+        infoTextView.font = UIFont.getFont(.book, size: 14)
+        infoTextView.isEditable = false
+        infoTextView.isScrollEnabled = false
+        view.addSubview(infoTextView)
+        
+        // fill with previously saved socials from backend
+        instaTextField.text = "@cndyhuang"
+        instaTextField.font = ._20CircularStdBook
+        instaTextField.backgroundColor = .white
+        instaTextField.layer.cornerRadius = 8
+        instaTextField.textColor = .textBlack
+        instaTextField.layer.shadowColor = UIColor(red: 0, green: 0, blue: 0, alpha: 0.15).cgColor
+        instaTextField.layer.shadowOffset = CGSize(width: 0, height: 2)
+        instaTextField.layer.shadowOpacity = 1
+        instaTextField.layer.shadowRadius = 4
+        view.addSubview(instaTextField)
+        
+        
+        let instaicon = UIImageView(frame: CGRect(x: 10, y: 0, width: 20, height: 20))
+        instaicon.image = UIImage(named: "instaicon")
+        let instaViewL = UIView(frame: CGRect(x: 0, y: 0, width: 40, height: 20))
+        instaViewL.addSubview(instaicon)
+        instaTextField.leftView = instaViewL
+        instaTextField.leftViewMode = .always
+        
+        let instaViewR = UIView(frame: CGRect(x: 0, y: 0, width: 10, height: 20))
+        instaTextField.rightView = instaViewR
+        instaTextField.rightViewMode = .always
+        
+        // fill with previously saved socials from backend
+        fbTextField.text = "http://www.facebook.co..."
+        fbTextField.font = ._20CircularStdBook
+        fbTextField.textColor = .textBlack
+        fbTextField.backgroundColor = .white
+        fbTextField.layer.cornerRadius = 8
+        fbTextField.layer.shadowColor = UIColor(red: 0, green: 0, blue: 0, alpha: 0.15).cgColor
+        fbTextField.layer.shadowOffset = CGSize(width: 0, height: 2)
+        fbTextField.layer.shadowOpacity = 1
+        fbTextField.layer.shadowRadius = 4
+        view.addSubview(fbTextField)
+        
+        let fbicon = UIImageView(frame: CGRect(x: 10, y: 0, width: 20, height: 20))
+        fbicon.image = UIImage(named: "facebookicon")
+        let facebookViewL = UIView(frame: CGRect(x: 0, y: 0, width: 40, height: 20))
+        facebookViewL.addSubview(fbicon)
+        fbTextField.leftView = facebookViewL
+        fbTextField.leftViewMode = .always
+        
+        let facebookViewR = UIView(frame: CGRect(x: 0, y: 0, width: 10, height: 20))
+        fbTextField.rightView = facebookViewR
+        fbTextField.rightViewMode = .always
```

Comment:
what are these right views for?

Reply:
there is a little padding on the right of the textfield in the figma